homeassistant/components/twilio/__init__.py

```diff
@@ -0,0 +1,80 @@
+"""
+Support for Twilio.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/twilio/
+"""
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import CONF_WEBHOOK_ID
+from homeassistant.helpers import config_entry_flow
+
+REQUIREMENTS = ['twilio==5.7.0']
+
+DOMAIN = 'twilio'
+
+API_PATH = '/api/{}'.format(DOMAIN)
+
+CONF_ACCOUNT_SID = 'account_sid'
+CONF_AUTH_TOKEN = 'auth_token'
+
+DATA_TWILIO = DOMAIN
+DEPENDENCIES = ['webhook']
+
+RECEIVED_DATA = '{}_data_received'.format(DOMAIN)
+
+CONFIG_SCHEMA = vol.Schema({
+    vol.Optional(DOMAIN): vol.Schema({
+        vol.Required(CONF_ACCOUNT_SID): cv.string,
+        vol.Required(CONF_AUTH_TOKEN): cv.string,
+        vol.Optional(CONF_WEBHOOK_ID): cv.string,
```

Comment:
Now that I think about it, we probably don't want to support the webhook id being set by the yaml config...